lib/repositories/usecases/GetAllLiquidityInfoUseCase.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:ax_dapp/pages/pool/MyLiqudity/models/MyLiquidityItemInfo.dart';
+import 'package:ax_dapp/repositories/subgraph/SubGraphRepo.dart';
+import 'package:ax_dapp/repositories/subgraph/usecases/SubgraphError.dart';
+import 'package:ax_dapp/service/BlockchainModels/LiquidityPosition.dart';
+import 'package:decimal/decimal.dart';
+import 'package:fpdart/fpdart.dart';
+
+class GetAllLiquidityInfoUseCase {
+  final SubGraphRepo _graphRepo;
+
+  GetAllLiquidityInfoUseCase(this._graphRepo);
+
+  Future<Either<Success, SubgraphError>> fetchAllLiquidityTokensInfoForWallet(
```

Comment:
this sounds kind of awkward maybe 
`fetchInfoForAllLiquidityToken` or `fetchAllLiquidityTokenDetails`

Reply:
Changed it to fetchAllLiquidityPositions.